hasktorch/src/Torch/Functional.hs

```diff
@@ -251,89 +238,77 @@ matmul
 exp 
     :: Tensor -- ^ input
     -> Tensor -- ^ output
 exp t = unsafePerformIO $ (cast1 ATen.exp_t) t
 
--- | log1p
--- Returns a new tensor with the natural logarithm of (1 + input).
+-- | Returns a new tensor with the natural logarithm of (1 + input).
 log1p 
     :: Tensor -> Tensor
 log1p t = unsafePerformIO $ (cast1 ATen.log1p_t) t
 
--- | log2
--- Returns a new tensor with the logarithm to the base 2 of the elements of input.
+-- | Returns a new tensor with the logarithm to the base 2 of the elements of input.
 log2 
     :: Tensor -- ^ input
     -> Tensor -- ^ output
 log2 t = unsafePerformIO $ (cast1 ATen.log2_t) t
 
--- | log10
--- Returns a new tensor with the logarithm to the base 10 of the elements of input.
+-- | Returns a new tensor with the logarithm to the base 10 of the elements of input.
 log10 
     :: Tensor -- ^ input
     -> Tensor -- ^ output
 log10 t = unsafePerformIO $ (cast1 ATen.log10_t) t
 
--- | pow
--- Takes the power of each element in input with exponent and returns a tensor with the result.
+-- | Takes the power of each element in input with exponent and returns a tensor with the result.
 pow 
  :: Scalar a 
  => Tensor -- ^ input
  -> a -- ^ exponent
  -> Tensor -- ^ output
 pow t s = unsafePerformIO $ (cast2 ATen.pow_ts) t s
 
--- | powt
--- Takes the power of each element in input with exponent and returns a tensor with the result.
+-- | Takes the power of each element in input with exponent and returns a tensor with the result.
 -- Exponent is a tensor with the same number of elements as input.
 powt
     :: Tensor -- ^ input
     -> Tensor -- ^ exponent
     -> Tensor -- ^ output
 powt t t' = unsafePerformIO $ (cast2 ATen.pow_tt) t t'
 
--- | relu
--- Applies the rectified linear unit function element-wise.
+-- | Applies the rectified linear unit function element-wise.
 relu 
     :: Tensor -- ^ input
     -> Tensor -- ^ output
 relu t = unsafePerformIO $ (cast1 ATen.relu_t) t
 
--- | selu
--- Applies element-wise, \text{SELU}(x) = scale * (\max(0,x) + \min(0, \alpha * (\exp(x) - 1)))SELU(x)=scale∗(max(0,x)+min(0,α∗(exp(x)−1))) , with \alpha=1.6732632423543772848170429916717α=1.6732632423543772848170429916717 and scale=1.0507009873554804934193349852946scale=1.0507009873554804934193349852946 .
+-- | Applies element-wise, \text{SELU}(x) = scale * (\max(0,x) + \min(0, \alpha * (\exp(x) - 1)))SELU(x)=scale∗(max(0,x)+min(0,α∗(exp(x)−1))) , with \alpha=1.6732632423543772848170429916717α=1.6732632423543772848170429916717 and scale=1.0507009873554804934193349852946scale=1.0507009873554804934193349852946 .
 selu 
     :: Tensor -- ^ input
     -> Tensor -- ^ output
 selu t = unsafePerformIO $ (cast1 ATen.selu_t) t
 
--- | celu
--- Applies element-wise, \text{CELU}(x) = \max(0,x) + \min(0, \alpha * (\exp(x/\alpha) - 1))CELU(x)=max(0,x)+min(0,α∗(exp(x/α)−1)) .
+-- | Applies element-wise, \text{CELU}(x) = \max(0,x) + \min(0, \alpha * (\exp(x/\alpha) - 1))CELU(x)=max(0,x)+min(0,α∗(exp(x/α)−1)) .
```

Comment:
not sure if latex renders correctly. it's ok now, but be prepared to revise once this is published at hasktorch.github.io

Reply:
Sure.